test/cmdlineTests/yul_function_name_clashes_different_params/output

```diff
@@ -4,90 +4,56 @@
 Pretty printed source:
 object "object" {
     code {
-        let a
-        let b
         {
-            function z() -> y
-            { y := calldataload(0) }
+            let a
+            let b
             a := z()
+            b := z_1(0x70)
+            sstore(a, b)
         }
-        {
-            function z(r) -> y
-            { y := calldataload(r) }
-            b := z(0x70)
-        }
-        sstore(a, b)
+        function z() -> y
+        { y := calldataload(0) }
+        function z_1(r) -> y
+        { y := calldataload(r) }
     }
 }
```

Comment:
Note that the new settings also apply to pure Yul compilation and the `--ir` output. Otherwise two-step compilation and assembling would no longer be equivalent to `--via-ir` in a single step.

Reply:
This is no longer a problem. Now `msize` in pure Yul without optimization disables Yul optimizer as well.